From: Jeremy Linton <jeremy.linton@arm.com>
Date: Wed, 4 May 2022 12:30:34 -0500
Subject: [PATCH 14/24] lscpu: Add Snapdragon parts

QC updated the kernel cputype.h to include the Kyro 4xx and 3xx Silver
part numbers's. Lets sync that commit and naming.

Signed-off-by: Jeremy Linton <jeremy.linton@arm.com>
---
 sys-utils/lscpu-arm.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/sys-utils/lscpu-arm.c b/sys-utils/lscpu-arm.c
index 86e5ea7..57b4bba 100644
--- a/sys-utils/lscpu-arm.c
+++ b/sys-utils/lscpu-arm.c
@@ -129,6 +129,9 @@ static const struct id_part qcom_part[] = {
     { 0x211, "Kryo" },
     { 0x800, "Falkor V1/Kryo" },
     { 0x801, "Kryo V2" },
+    { 0x803, "Kryo 3XX Silver" },
+    { 0x804, "Kryo 4XX Gold" },
+    { 0x805, "Kryo 4XX Silver" },
     { 0xc00, "Falkor" },
     { 0xc01, "Saphira" },
     { -1, "unknown" },
